crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -189,9 +188,39 @@ impl GlobalTransform {
         (self.0.matrix3 * extents).length()
     }
 
-    /// Transforms the given `point`, applying shear, scale, rotation and translation.
+    /// Transforms the given point from local space to global space, applying shear, scale, rotation and translation.
+    ///
+    /// It can be used like this:
+    ///
+    /// ```
+    /// # use bevy_transform::prelude::{GlobalTransform};
+    /// # use bevy_math::prelude::Vec3;
+    /// let global_transform = GlobalTransform::from_xyz(1., 2., 3.);
+    /// let local_point = Vec3::new(1., 2., 3.);
+    /// let global_point = global_transform.transform_point(local_point);
+    /// assert_eq!(global_point, Vec3::new(2., 4., 6.));
+    /// ```
     ///
-    /// This moves `point` into the local space of this [`GlobalTransform`].
+    /// ```
+    /// # use bevy_transform::prelude::{GlobalTransform};
+    /// # use bevy_math::Vec3;
+    /// let global_point = Vec3::new(2., 4., 6.);
+    /// let global_transform = GlobalTransform::from_xyz(1., 2., 3.);
+    /// let local_point = global_transform.affine().inverse().transform_point3(global_point);
+    /// assert_eq!(local_point, Vec3::new(1., 2., 3.))
+    /// ```
+    ///
+    /// To apply shear, scale, and rotation WITHOUT applying translation, different functions are available:
```

Comment:
```suggestion
    /// To apply shear, scale, and rotation *without* applying translation, different functions are available:
```

maybe nicer typographically (but I don't really mind)